### PR TITLE
Multiarch support

### PR DESCRIPTION
**Pull request by roobre**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,33 +1,27 @@
...
-    ca-certificates \
+    tini ca-certificates \
     && addgroup -g 2000 nri-kube-events \
     && adduser -D -H -u 1000 -G nri-kube-events nri-kube-events
-USER nri-kube-events
-
-COPY --from=build-env /src/nri-kube-events .
 EXPOSE 8080
 
+ADD --chmod=755 bin/nri-kube-events-${TARGETOS}-${TARGETARCH} ./
```
Comment (paologallinaharbur) [inline Dockerfile:18]: does it work locally?

Comment (roobre) [reply Dockerfile:18]: Yes, both locally and afaik in the CI as well. Why do you ask?

**Review by paologallinaharbur — APPROVED** [review]
LGTM just keep in mind to reenable commented out feature before merging

**Review by paologallinaharbur — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -1,33 +1,27 @@
...
+RUN mv nri-kube-events-${TARGETOS}-${TARGETARCH} nri-kube-events
```
Comment (gsanchezgavier) [inline Dockerfile:19]: just out of curiosity, couldn't the binary be renamed in the ADD line above?

File: Makefile [hunk 3]
```diff
@@ -33,7 +46,8 @@ lint: $(TOOLS_DIR)/golangci-lint
 
 compile:
 	@echo "=== $(INTEGRATION) === [ compile ]: Building $(INTEGRATION)..."
-	@go build -o $(BUILD_TARGET) ./cmd/nri-kube-events
+	go mod download
```
Comment (gsanchezgavier) [inline Makefile:49]: why we pre-download the modules ? Also just out of curiosity.

**Review by gsanchezgavier — APPROVED** [review]

File: Dockerfile (hunk 2 above)
Comment (roobre) [reply Dockerfile:19]: Because docker is ~~stupid~~ limited and that doesn't work :(

In my local machine, doing
`ADD --chmod=755 bin/nri-kube-events-${TARGETOS}-${TARGETARCH} /app/nri-kube-events` works fine. However, on the CI this will result in the binary ending up in `/app/nri-kube-events/kube-events-linux-amd64`. Reason for this is likely that this was changed/fixed in some docker version that the GHA ubuntu env doesn't have. This happened with both `COPY` and `ADD`, so I went for the quick hack.

File: Makefile (hunk 3 above)
Comment (roobre) [reply Makefile:49]: Nice point, not really sure why I added it here.

Comment (roobre) [reply Makefile:49]: Ok I just remembered, it is needed if the build tree is in the GOPATH, as `go build` doesn't download modules automatically in that case.

File: .dockerignore [hunk 4]
```diff
@@ -1,4 +1,3 @@
-bin/*
```
Comment (paologallinaharbur) [inline .dockerignore:1]: is this change intended?

File: cmd/nri-kube-events/main.go [hunk 5]
```diff
@@ -27,7 +27,7 @@ import (
 
 const (
 	integrationName    = "nri-kube-events"
-	integrationVersion = "1.0.0"
```
Comment (paologallinaharbur) [inline cmd/nri-kube-events/main.go:30]: can we make this automatic with build flags? https://github.com/newrelic/nri-kube-events/issues/26

**Review by paologallinaharbur — APPROVED** [review]

File: cmd/nri-kube-events/main.go (hunk 5 above)
Comment (paologallinaharbur) [reply cmd/nri-kube-events/main.go:30]: the is also eventRouterVersion for some reason that is set statically

File: .dockerignore (hunk 4 above)
Comment (roobre) [reply .dockerignore:1]: Yes! You might remember me losing my mind because the `ADD` command not working because of this dumb line.

File: cmd/nri-kube-events/main.go (hunk 5 above)
Comment (roobre) [reply cmd/nri-kube-events/main.go:30]: I bumped those numbers as a part of the version bump, but I'm afraid that making them dynamic is not as easy as copy-pasting from one of our other repos since we don't use goreleaser for this one. I'd rather descope that and leave it for another PR.

File: .dockerignore (hunk 4 above)
Comment (paologallinaharbur) [reply .dockerignore:1]: ups, it is dockerignore, not gitignore, then yes I remember 😅

File: cmd/nri-kube-events/main.go (hunk 5 above)
Comment (paologallinaharbur) [reply cmd/nri-kube-events/main.go:30]: :fine for me!

**Review by paologallinaharbur — APPROVED** [review]
